test/verify/check-pages

```diff
@@ -216,6 +216,62 @@ OnCalendar=daily
         b.wait_visible('#password-group')
         b.wait_text("#password-group > label", "Passwort")
 
+        # Test all languages
+        # Test that pages do not oops and that locale is valid
+
+        def line_sel(i):
+            if m.image == "rhel-7-6-distropkg":
+                return '.terminal div:nth-child(%d)' % i
+            else:
+                return '.terminal .xterm-accessibility-tree div:nth-child(%d)' % i
```

Comment:
FWIW,  I think it's fine  to just skip that test on *-distropkg, atomics, and others -- testing this on one release (like fedora) seems fine. Feel free to make it a new test case, then it's easier to do so.

Reply:
I am going to tests this only on F29 and F30. I don't want to create a separate test case as booting machine takes some time and this is rather slow anyway.
Pushed a edited version.